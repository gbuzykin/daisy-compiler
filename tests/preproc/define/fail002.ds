#define MACRO# include "1.ds"

const a = MACRO ;