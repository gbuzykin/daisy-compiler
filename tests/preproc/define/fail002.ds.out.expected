./preproc/define/fail002.ds:3:1: debug: token
 3 | const a = MACRO ;
   | ^~~~~
./preproc/define/fail002.ds:3:7: debug: id: a
 3 | const a = MACRO ;
   |       ^
./preproc/define/fail002.ds:3:9: debug: token
 3 | const a = MACRO ;
   |         ^
./preproc/define/fail002.ds:3:11: error: unexpected token
 3 | const a = MACRO ;
   |           ^~~~~
./preproc/define/fail002.ds:1:14: note: expanded from macro `MACRO`
 1 | #define MACRO# include "1.ds"
   |              ^
./preproc/define/fail002.ds:3:17: debug: token
 3 | const a = MACRO ;
   |                 ^
./preproc/define/fail002.ds: info: warnings 0, errors 1
